### PR TITLE
Audit its own channel link on video description

### PR DESCRIPTION
**Pull request by kangkyu**
Add `has_youtube_association?` method

### CODE REVIEW

File: lib/yt/audit.rb [hunk 1]
```diff
@@ -29,5 +29,15 @@ def self.has_subscribe_annotations?(video_id)
         annotation.link && annotation.link[:type] == :subscribe
       end
     end
+
+    # Audit youtube association of a video
+    # @param [String] video_id the video to audit.
+    # @return [Boolean] if the video description has link to its own channel.
+    def self.has_youtube_association?(video_id)
```
Comment (claudiofullscreen) [inline lib/yt/audit.rb]: Is this the name that is used in the report? Can we come up with a better name?

In my opinion, "has YouTube association" is too generic to understand what this method does.

Comment (kangkyu) [inline lib/yt/audit.rb]: Yes, the name was used in the report. How about we name the method `has_link_to_own_channel?` and then on the Audit first page "YouTube Association: 10 out of 10"

Comment (claudiofullscreen) [inline lib/yt/audit.rb]: Yes, that's better.

On Tue, Feb 16, 2016 at 1:48 PM, Kang-Kyu Lee <notifications@github.com>
wrote:

> In lib/yt/audit.rb
> <https://github.com/Fullscreen/yt-audit/pull/6#discussion_r53084606>:
>
> > @@ -29,5 +29,15 @@ def self.has_subscribe_annotations?(video_id)
> >          annotation.link && annotation.link[:type] == :subscribe
> >        end
> >      end
> > +
> > +    # Audit youtube association of a video
> > +    # @param [String] video_id the video to audit.
> > +    # @return [Boolean] if the video description has link to its own channel.
> > +    def self.has_youtube_association?(video_id)
>
> Yes, the name was used in the report. How about we name the method
> has_link_to_own_channel? and then on the Audit first page "YouTube
> Association: 10 out of 10"
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/Fullscreen/yt-audit/pull/6/files#r53084606>.
>



-- 


 Claudio Baccigalupo  | Manager, Software Engineering  |  Fullscreen, Inc.
<http://www.fullscreen.com>

**Comment by claudiofullscreen** [comment]
:+1: you can release, @kangkyu :tada:

**Comment by kangkyu** [comment]
Thank you. released!

**Pull request merged**